R/arima.R

```diff
@@ -558,7 +555,16 @@ glance.ARIMA_exploratory <- function(x, pretty.name = FALSE, ...) { #TODO: add t
   # forecast::accuracy(m) seems to be returning NaNs.
   # df <- data.frame(AIC=m$aic, BIC=m$bic, AICc=m$aicc, as.list(forecast::arimaorder(m)), forecast::accuracy(m))
   # TODO: migrate out from forecast package.
-  df <- data.frame(AIC=m$aic, BIC=m$bic, AICc=m$aicc, as.list(forecast::arimaorder(m)))
+
+  # Extract orders from model.
+  order <- m$arma[c(1, 6, 2, 3, 7, 4, 5)]
```

Comment:
Can you leave a comment for those lists? (1,6,2,3... and p,d,q,....) 

Also, please leave a comment why you can tell this is seasonal or not by this calculation.